common/environment/environment.hpp

```diff
@@ -111,6 +116,12 @@ interface IConstDropZoneInfo : extends IConstEnvBase
     virtual IStringVal & getUMask(IStringVal & str) const = 0;
 };
 
+interface  IConstDropZoneInfoIterator : extends IIteratorOf<IConstDropZoneInfo>
+{
+    virtual unsigned count() = 0;
+    virtual IConstDropZoneInfo * ptr() = 0;
```

Comment:
ptr() should be removed, IIterator* methods don't need it. query() is sufficient and standard.